packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -286,7 +286,12 @@ export class SummarizerNode implements IRootSummarizerNode {
         );
         return { latestSummaryUpdated: true, wasSummaryTracked: false, snapshot: snapshotTree };
     }
-
+/**
```

Comment:
Please check with @NicholasCouri if its safe to pick this change here.

Reply:
looks good to me :)